recipes/xz/5.2.5/meta.yaml

```diff
@@ -0,0 +1,90 @@
+{% set version = "5.2.5" %}
+{% set sha256 = "5117f930900b341493827d63aa910ff5e011e0b994197c3b71c08a20228a42df" %}
+
+package:
+  name: xz-pkg
+  version: "{{ version }}"
+
+about:
+  home: https://tukaani.org/xz/
+  license: Various
+  summary: XZ Utils is free general-purpose data compression software.
+
+build:
+  number: 4
```

Comment:
Build number should be `0`

Reply:
agree  - but I 've mucked up this PR - I'll rebase

Reply:
I'd already accepted that into `devel` - @kjsanger still reset? OR leave until next time?

Reply:
If the package hasn't been pushed to a channel, we can fix it. If by `devel` you mean the channel rather than the git branch, then we should leave it and increment from there.